src/legacy/core_plugins/kibana/public/discover/components/field_chooser/_field_chooser.scss

```diff
@@ -2,3 +2,10 @@
   color: $euiColorMediumShade;
   margin-left: $euiSizeS !important;
 }
+
+/*
+  Fixes EUI bug https://github.com/elastic/eui/issues/1749
+*/
+.stringProgressBarTooltip__anchor {
```

Comment:
Please change the prefix here from `string` to `dsc`. Throughout Kibana, we use these 3-letter prefixes within each app as demonstrated by the other rule in this file (above).

Reply:
@ryankeairns done